salt/metalk8s/salt/master/files/master-99-metalk8s.conf.j2

```diff
@@ -45,6 +45,7 @@ external_auth:
     storage-operator:
       - '*':
         - 'disk.dump'
+        - 'metalk8s_volumes.device_name'
```

Comment:
Weirdly enough, it was working fine in 2.5 without that, but was failing in 2.6 with a 401 error from SaltAPI (as expected, since the permission was missing).

Reply:
That's a bug, here's a ticket to investigate: #2634